torch/csrc/api/src/nn/modules/activation.cpp

```diff
@@ -170,8 +170,8 @@ void Softmax2dImpl::pretty_print(std::ostream& stream) const {
 }
 
 Tensor Softmax2dImpl::forward(const Tensor& input) {
-  TORCH_CHECK(input.dim() == 4, "Softmax2d requires a 4D tensor as input");
-  return F::detail::softmax(input, /*dim=*/1, c10::nullopt);
+  TORCH_CHECK(input.dim() == 4 or input.dim() == 3, "Softmax2d requires a 3D or 4D tensor as input");
```

Comment:
@thomasjpfan FYI we're getting a bunch of internal failures on Windows due to this usage of "or" instead of "||" that slipped in here :p